index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+    <title>Youtube Video Player</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css" />
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <nav>
+                <img id="menu" src="images/bars.png" alt="menu">
+                <img id="youtube_logo" src="images/Youtube_Logo.jpg" alt="youtube_logo">
+                <div>
+                    <input type="text" placeholder="Search">
+                    <button><i class="fas fa-search"></i></button>
+                </div>
+                <img id="upload" src="images/arrows.png" alt="upload">
+                <button id="signin">Sign in</button>
+            </nav>
+            <div id="language">
+                <p>Choose your language<span>x</span></p>
+            </div>
+            <div id="learn_more">
+                <p>You are viewing YouTube in English (US). You can change this preference below <a href="">Learn
+                        more</a></p>
+            </div>
+        </header>
+
+        <div class="video_screen">
+            <video width="640" height="360" controls>
```

Comment:
This should be an `iframe` because your goal is to embed a youtube video on your page.

Reply:
Hello Santiago, thanks a lot for the feedback. We missed the video on embedding videos in TOP. We will pay more attention to details from now on.